tex/artikel/Muensterraetsel.tex

```diff
@@ -182,12 +182,12 @@ \subsection{Überlebenstraining für Physiker in Münster}
 	\end{fibelanswer}
 	\label{itm:rätsel_bruce}
 	% XXX Jedes Jahr prüfen, ob der Preis noch stimmt!
-	\item Wie viel kostet ein vegetarischer Mensa-Burger?
+	\item Warum sprudelt es an heißen Tagen im Aasee?
 	\begin{fibelanswer}
-		\item \SI{2,10}{\euro}
-		\item \SI{1,95}{\euro}
-		\item \SI{1,50}{\euro}
-		\item Igitt
+		\item Belüftungsgeräte, die sauerstoffreiche und sauerstoffarme Wasserschichten für die Fische vermischen
+		\item Um Besucher davon abzuhalten baden zugehen
+		\item Kunst aus dem Jahre 2017 im Rahmen der Skulptur Projekte
+		\item Faulgase, die bei der Hitze aufsteigen
 	\end{fibelanswer}
 	\label{itm:rätsel_burger}
```

Comment:
```suggestion
	\label{itm:rätsel_aasee}
```
Der Übersicht halber. Müsste auch in https://github.com/HannahNeuwirth/ersti-fibel/blob/patch-4/tex/artikel/Notizen_Loesungen.tex aktualisiert werden.